src/UI/Series/Details/InfoViewTemplate.hbs

```diff
@@ -55,6 +55,10 @@
             {{#if_eq seasonNumber compare="-1"}}
                 <span class="label label-default">{{title}}</span>
             {{/if_eq}}
+
+            {{#if_eq sceneSeasonNumber compare="-1"}}
```

Comment:
No so sure about that, what about Spartacus?
Blood.and.Sand.S01 -> tvdb S01
Gods.of.the.arena.S01 -> tvdb S00
Vengeance.S02 -> tvdb S02
War.of.the.Damned.S03 -> tvdb S03

The special case being Gods.of.the.Arena.